source/ghost-text.js

```diff
@@ -68,6 +68,12 @@ class AdvancedTextWrapper {
 }
 
 function wrapField(field) {
+	const monaco = field.closest('.monaco-editor, .editor-widget');
+	if (monaco && monaco.matches('.monaco-editor')) {
+		const visualElement = document.querySelector('.lines-content.monaco-editor-background');
```

Comment:
@fregante can this be a follow up? didn't think about if there were multiple monaco editors. Spent some time trying to figure out how to determine which monaco editor is the active or focused one? there's no `getActiveEditor` or `getFocusedEditor`? Do you have any suggestion/idea?

Reply:
In my original comment you'll find some suggestions:

https://github.com/fregante/GhostText/issues/140#issuecomment-1221505023

It's the `.find` line

Reply:
The "visual element" is the element used to show the glow on. Since this only selects the first element, you can see here here that it detected 3 elements, but only the first one is highlighted:

<img width="723" alt="Screenshot 2" src="https://user-images.githubusercontent.com/1402241/215011476-ad6362be-30aa-4c81-a570-13d83d5e5606.png">